libobs/obs-av1.h

```diff
@@ -30,9 +39,9 @@ EXPORT void obs_extract_av1_headers(const uint8_t *packet, size_t size,
 				    size_t *new_packet_size,
 				    uint8_t **header_data, size_t *header_size);
 
-EXPORT void metadata_obu_itu_t35(const uint8_t *itut_t35_buffer,
```

Comment:
This symbol is public, please preserve the old function it can be implemented with the new function trivially.

Reply:
Thank you @kkartaltepe. I'll fix this up. Appreciate your review.

Reply:
I've update this with the latest git push. I put the original export back and wrapped the call to the new `metadata_obu()` function.